docs/tutorial/prerequisites.md

```diff
@@ -0,0 +1,49 @@
+Spilo combines [Patroni](https://github.com/zalando/Patroni) with [stups](https://stups.io). Running Spilo therefore
+requires that you have an Amazon VPC with Stups installed. For more details,
+see the [Stups Installation Guide](http://docs.stups.io/en/latest/installation/)
+
+## Stups
+
+To be able to deploy and troubleshoot Spilo, you need the stups tooling installed locally:
+```bash
+$ sudo pip3 install --upgrade stups
+```
+
+If you need different installation methods, run into issues, or want more details on how to install the stups tooling
+locally, we refer you to the [Stups User's Guide](http://docs.stups.io/en/latest/user-guide/index.html)
+
+You can test your setup by logging in using mai and then listing the already running instances using senza,
+example session:
+
+```bash
+$ mai login
+Please enter password for encrypted keyring:
+Authenticating against https://aws.example.com.. OK
+Assuming role AWS Account 1234567890 (example-team): SomeRoleName.. OK
+Writing temporary AWS credentials.. OK
+```
+```bash
+$ senza list
```

Comment:
It is not a prerequisite to have spilo running, that's why I left it out. It is a prerequisite to have etcd-cluster running, so I could add that?